runtime/kusama/src/lib.rs

```diff
@@ -2478,3 +2502,50 @@ mod remote_tests {
 		});
 	}
 }
+
+mod init_state_migration {
```

Comment:
Would this be better off living in `frame-support/migrations` or somewhere else where it can be reused in different runtimes?

Reply:
No. This is a chain specific migration/initialization that doesn't belong somewhere in FRAME.